client/state/sites/seo/schema.js

```diff
@@ -0,0 +1,15 @@
+export const titleFormatSchema = {
+	type: 'object',
+	properties: {
+		id: {
```

Comment:
By default `additionalProperties` is always true, so we allow extra unexpected properties on an object. This is helpful when there's an additive change to an api. 

When using patternProperties, we set this value to false, so we can actually limit what we want, in this case, siteId keys.